### DISCUSSION TITLE
Mars v1.50 Major Release

### DISCUSSION THREAD

**Discussion by RedFox20** [TeamStarDrive/StarDrive#125] [Announcements]
**Installation Instructions**
- _Do not install this release on your Stardrive Steam folder_
- Download link: [StarDrive_BlackBox_Mars_1.50.msi](https://github.com/TeamStarDrive/StarDrive/releases/download/mars-release-1.50/StarDrive_BlackBox_Mars_1.50.msi)
- Install the game to a modifiable location such as the default C:/Games/StarDrivePlus. DO NOT INSTALL INTO C:/Program Files/ !!!
- Start the game from Windows Start Menu or from your desktop "StarDrive BlackBox"
- For bug reports join our [Discord Discussion](https://discord.gg/dfvnfH4)
- Always include your blackbox.log, which is located at C:/Games/StarDrivePlus/blackbox.log

**Major Features of 1.50**

- Mining Ops! Some Gas Giants can be mined for their Exotic resources, which will yield empire wide bonuses. To mine the resources, you need to deploy mining stations.

- Mining Stations will launch small mining ships that will go into lower orbit to mine the resources. Mining speed depends on how rich the Gas giant is. Mining ships will bring the goods aboard the mining station and then a refining process will start. The refining process consumes food, which can be handled by your freighter fleets. Excess goods can be stored in a wide empire storages (this is abstract), which can be enlarged. The refining process creates the refined resources, there is a conversion rate, usually 0.5 at start (techs can improve it). So if you can refine 4 raw resources per turn, 2 will come out of the process.

- New techs can improve imperial storage size, mining ships speed and refine ratio.

- There is a new Exotic Resources window where you can see current bonuses (short key - M). The Mining Ops feature might need balance and be unstable. You can disable mining ops in the rule options screen, before starting a new game.

- New Audio System with reduced memory usage and advanced modding support. This is a major change towards having a 64-bit StarDrive in the future.

- Various fixes in hangar ships, research stations, repair and game UI.

<hr /><em>This discussion was created from the release <a href='https://github.com/TeamStarDrive/StarDrive/releases/tag/mars-release-1.50'>Mars v1.50 Major Release</a>.</em>

**Comment by Whistar** [comment]
Thanks a lot for the continued updates. This one looks interesting, finally something to do with the gas planets!!
Q: With Cybernatic trait, one can no longer research to Assimilate, Terraform, Hive cities or planetary defenses. Was that on purpose?

**Reply by gkapulis** [reply]
Are you playing with Combined Arms or Blackbox Plus?

**Reply by Whistar** [reply]
yes, both
